Sum of digits of a number.py

```diff
@@ -1,7 +1,13 @@
-q=0
-n=int(input("Enter Number: "))
-while(n>0):
- r=n%10
- q=q+r
- n=n//10
+q=0   # Initially we assigned 0 to "q", to use this variable for the summation purpose below.
+    # The "q" value should be declared before using it(mandatory). And this value can be changed later.
+
+n=int(input("Enter Number: "))   # asking user for input
+while n>0:         # Until "n" is greater than 0, execute the loop. This means that until all the digits of "n" got extracted.
```

Comment:
But buddy, I love your approach of understanding as well at the same time contributing to the repository.
It is like contributing + understanding the program, simultaneously.
I would too, try this approach.
But, it would be helpful, if you try to improve the execution and logic and readability of the program.

Reply:
Thanks for your suggestion NitkarshChourasia. I'll follow this approach.

Reply:
@sarayusreeyadavpadala Do check my pull requests also.
Let's grow together.
I will always @sarayusreeyadavpadala assign you like this.
You too, try to assign me.
Because coding is not supposed to be lonely activity, but a fun and a social one.
That is called pair programming, I guess.
Right?
Let's do it.
I will post my programs.
See, what things I do.
Basically what I follow is:
I comment it.
Then, write down my basic requirements like,

"""
Requirements:
- DRY
- KISS
- OOP
- Function / Class
- snake_case
- Docstring 
- Comment
- unittest
- type hinting
- type checking
- Good logic (subjective)
- Readablility (subjective)
- Proper formatting using black
- Optimising the imports.

Etc...
This is what I like **AT LEAST** do...

Come on, all the best.

Reply:
#Let'sGrowTogether @NitkarshChourasia

Reply:
@sarayusreeyadavpadala review my code, I've changed it.

Reply:
@NitkarshChourasia view my code, I've changed it.

Reply:
# Yes.